testblas/level1double.go

```diff
@@ -1174,6 +1214,105 @@ var DoubleTwoVectorCases = []DoubleTwoVectorCase{
 			},
 		},
 	},
+	{
+		Name:  "Empty",
+		X:     []float64{},
+		Y:     []float64{},
+		Incx:  1,
+		Incy:  1,
+		N:     0,
+		Panic: false,
+		DaxpyCases: []DaxpyCase{
+			{
+				Alpha: 2,
+				Ans:   []float64{},
+			},
+		},
+		DrotCases: []DrotCase{
+			{
+				C:    math.Cos(25 * math.Pi / 180),
+				S:    math.Sin(25 * math.Pi / 180),
+				XAns: []float64{},
+				YAns: []float64{},
+			},
+		},
+		DrotmCases: []DrotmCase{
+			{
+				P: blas.DrotmParams{
+					Flag: blas.Rescaling,
+					H:    [4]float64{0.9, 0.1, -0.1, 0.5},
+				},
+				XAns: []float64{},
+				YAns: []float64{},
+			},
+		},
+	},
+	{
+		Name:  "EmptyStationary",
```

Comment:
Instead of this, could we have "IncZeroX" and "IncZeroY" to test that the functions fail when each are individually zero? If a routine only checks that IncX is zero, it will panic even though incY = 0.